README.md

```diff
@@ -126,21 +145,33 @@ MLRun has many code examples and tutorial Jupyter notebooks with embedded docume
   paragraph, update the heading levels, add a `top` anchor, and remove the
   "Back to quick-start TOC" links (leaving only the "Back to top" links). -->
 
-- [Basic Components](#basic-components)
-- [Managed and Portable Execution ](#managed-and-portable-execution)
-- [Automated Code Deployment and Containerization](#auto-parameterization-artifact-tracking-n-logging)
-- [Using Hyperparameters for Job Scaling](#using-hyperparameters-for-job-scaling)
-- [Automated Code Deployment and Containerization](#auto-code-deployment-n-containerization)
-- [Build and run function from a remote IDE using the CLI](examples/remote.md)
-- [Running an ML Workflow with Kubeflow Pipelines](#run-ml-workflow-w-kubeflow-pipelines)
-- [Viewing Run Data and Performing Database Operations](#db-operations)
-  - [The MLRun Dashboard](#mlrun-ui)
-  - [MLRun Database Methods](#mlrun-db-methods)
-- [Additional Information and Examples](#additional-info-n-examples)
-  - [Replacing Runtime Context Parameters from the CLI](#replace-runtime-context-param-from-cli)
-  - [Remote Execution](#remote-execution)
-- [Running an MLRun Service](#run-mlrun-service)
-  - [Using the MLRun CLI](#run-mlrun-service-cli)
+- [Benefits](#benefits)
+- [Components](#components)
+    - [In This Document](#in-this-document)
```

Comment:
In this document seems unrelated as a sub-title of components suggest to return it to the top as before plus changing it from `####` to `##`

Reply:
fixed